src/media/js/main.js

```diff
@@ -338,5 +338,12 @@ function(_) {
         throw new Error('Cancel navigation; deprecated client');
     });
 
+    // Remove paginated class from app lists if .loadmore goes away.
```

Comment:
Put this in app list js module. Need to add a header comment to main.js not to dump stuff here